packages/cli/lib/lib/webpack/webpack-client-config.js

```diff
@@ -103,6 +117,38 @@ async function clientConfig(env) {
 					},
 				].filter(Boolean)
 			),
+			...(env.sw
```

Comment:
this makes things a lot less readable

Reply:
The nested ternary? 

Yeah, sorry, can't remember why I did it like that. I'll try to fix that up tomorrow (3am here)

Reply:
Pushed an update, hopefully that's more readable.